src/main/java/seedu/address/model/team/Team.java

```diff
@@ -14,6 +14,9 @@
  */
 public class Team extends UniquePersonList {
 
+    public static final String TEAM_VALIDATION_REGEX = "\\p{Alnum}+";
```

Comment:
Same as mentioned below, will be using the validity checker in `TeamName.java` to check for team name validity.